experimental/Schemes/BlowupMorphism.jl

```diff
@@ -96,108 +96,236 @@ function strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
   return Z_trans, inc_Z_trans, pr_res
 end
 
+## THIS SHOULD GO TO mpolyquo-localization.jl
+## it is the localized and/or quotient counterpart to the MPolyIdeal-method
+function saturation(I::T,J::T) where T <: Union{ MPolyQuoIdeal, MPolyLocalizedIdeal, MPolyQuoLocalizedIdeal}
+  R = base_ring(I)
+  R == base_ring(J) || error("Ideals do no live in the same ring.")
+
+  I_sat = saturated_ideal(I)
+  J_sat = saturated_ideal(J)
+
+  I_result = Oscar.saturation(I_sat,J_sat)  ## why is saturation not exported from Rings/mpoly-ideal.jl?
+  return ideal(R,gens(I_result))
+end
+
+## WHERE SHOULD THIS GO?
+#@Markdown.doc """
+#  iterated_quotients(I::T, J::T, b::Int) where T <: MPolyAnyIdeal
+#
+#Return ``I:J^b`` and b, for a given natural number b.
+#Return this for the highest b such that ``J(I:J^b)== J^(b-1)``, if given b is zero.
+#
+#Internal function for weak and strict transform.
+#"""
+function iterated_quotients(I::T, J::T, b::Int) where T <: MPolyAnyIdeal
+  R = base_ring(I)
+  R == base_ring(J) || error("Ideals do no live in the same ring.")
+
+  Itemp = I
+  k = 0
+
+  while (b == 0 || k < b)
+    Itemp2 = quotient(Itemp, J)
+    if !issubset(Itemp, Itemp2 * J)
+       b == 0 || error("cannot extract J from I with multiplicity b")
+       break
+    end
+    Itemp = Itemp2
+    k = k+1
+  end
+
+  return Itemp,k
+end
+
+has_simplified_covering(C::Covering) = has_attribute(C, :simplified_covering)
+
+#@Markdown.doc """
+#  strict_transform(p::BlowupMorphism, I::IdealSheaf)
+#
+#For a 'BlowupMorphism'  ``p : Y → X`` and an IdealSheaf ''I'' on ''X'' return the
+#strict transform of ''I'' on ''Y''.
+#"""
```

Comment:
Why remove the docstrings? Do you change the behaviour?

Reply:
I only commented them out, because I somehow messed up the syntax (probably the accents). 
This is still a draft, so it will be corrected before making the PR ready for review.

Reply:
Problem with docstrings solved. They are now available